### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by wuhucheng** [ggsDing/Bi-SRNet#22]
The normal data set should include the training set verification set and the test set, and the final performance indicators are calculated on the test set, but the test set of the SECOND data set has no label, only two time-phase images. 
Are the indicators for measuring network performance obtained from the verification set?
 Or your complete data, that is, the label of the test set
![QQ截图20230228181343](https://user-images.githubusercontent.com/45914165/221824301-c5692592-fc62-4b79-86aa-167827e2e722.png)
![9](https://user-images.githubusercontent.com/45914165/221827028-8449e87d-3641-4377-89c9-626401f478b2.png)

The images contained in these two folders are the train and val in my folder. Do you mean to use the data in val.txt to test and output evaluation indicators?
If you need to divide the images contained in train.txt into training images and validation images, what are train and val images?

**Comment by wuhucheng** [comment]
And why Sek has a negative number during training.
![QQ截图20230227191514](https://user-images.githubusercontent.com/45914165/221549706-7a11bc29-4b83-46be-bb37-9a381f0ab92f.png)

**Comment by ggsDing** [comment]
Hi,
The test set is not provided with labels, so the validation sets are also used for testing.
As for SeK it is normal to have negative values, which means bad performance in  predicting the semantic categories.

**Comment by ggsDing** [comment]
Hi,
The test set is not provided with labels, so the validation sets are also used for testing.
As for SeK, negative values are normal, which means bad performance in predicting the semantic categories.

**Comment by ggsDing** [comment]
Hi,
The test set is not provided with labels, so the validation sets are also used for testing.
As for SeK it is normal to have negative values, which means bad performance in  predicting the semantic categories.
